charts/linstor-cluster/templates/storageclass.yaml

```diff
@@ -0,0 +1,22 @@
+{{- range .Values.storageClasses }}
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: {{ .name }}
```

Comment:
Since StorageClass names explicitly provided by values file, it is ok to not having release-name prefix for them.
Another words: keep it as is